lua/telescope/_extensions/file_browser/actions.lua

```diff
@@ -417,6 +417,37 @@ fb_actions.goto_cwd = function(prompt_bufnr)
   current_picker:refresh(finder, { reset_prompt = true, multi = current_picker._multi })
 end
 
+--- Change working directory of nvim to the selected file/folder in |builtin.file_browser|.
+---@param prompt_bufnr number: The prompt bufnr
+fb_actions.change_cwd = function(prompt_bufnr)
+  local picker = action_state.get_current_picker(prompt_bufnr)
+  local finder = picker.finder
+  local sel = fb_utils.get_selected_files(prompt_bufnr, true)[1]
```

Comment:
This should be

```lua
local entry = action_state.get_selected_entry().Path
```

as it shouldn't work through a multi-selection mechansim.

Reply:
Thanks :)